hack/update-generated-protobuf-dockerized.sh

```diff
@@ -41,7 +41,7 @@ fi
 
 gotoprotobuf=$(kube::util::find-binary "go-to-protobuf")
 
-APIROOTS=( ${1} )
+mapfile -t APIROOTS <<< "${1}"
```

Comment:
change reason:
```
In ./hack/update-generated-protobuf-dockerized.sh line 44:
APIROOTS=( ${1} )
           ^--^ SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.
```
Ref: https://github.com/koalaman/shellcheck/wiki/SC2206

Reply:
We can't use mapfile as it's not compatible with Bash v3. Prefer the read -a pattern. There are a number of examples of read-a in the scripts.

Reply:
Done!